ui/app/models/pki/crl.js

```diff
@@ -50,21 +62,37 @@ export default class PkiCrlModel extends Model {
     labelDisabled: 'OCSP responder APIs disabled',
     mapToBoolean: 'ocspDisable',
     isOppositeValue: true,
+    editType: 'ttl',
     helperTextEnabled: "Requests about a certificate's status will be valid for:",
     helperTextDisabled: 'Requests cannot be made to check if an individual certificate is valid.',
   })
   ocspExpiry;
 
-  // TODO follow-on ticket to add enterprise only attributes:
-  /*
-  @attr('boolean') crossClusterRevocation;
-  @attr('boolean') unifiedCrl;
-  @attr('boolean') unifiedCrlOnExistingPaths;
-  */
+  // enterprise only params
+  @attr('boolean', {
+    label: 'Cross-cluster revocation',
+    helpText:
+      'Enables cross-cluster revocation request queues. When a serial not issued on this local cluster is passed to the /revoke endpoint, it is replicated across clusters and revoked by the issuing cluster if it is online.',
+  })
+  crossClusterRevocation;
+
+  @attr('boolean', {
+    label: 'Unified CRL',
+    helpText:
+      'Enables unified CRL and OCSP building. This synchronizes all revocations between clusters; a single, unified CRL will be built on the active node of the primary performance replication (PR) cluster.',
+  })
+  unifiedCrl;
+
+  @attr('boolean', {
+    label: 'Unified CRL on existing paths',
+    helpText:
+      'If enabled, existing CRL and OCSP paths will return the unified CRL instead of a response based on cluster-local data.',
+  })
+  unifiedCrlOnExistingPaths;
 
   @lazyCapabilities(apiPath`${'id'}/config/crl`, 'id') crlPath;
 
   get canSet() {
-    return this.crlPath.get('canCreate') !== false;
+    return this.crlPath.get('canUpdate') !== false;
```

Comment:
Just out of curiosity why was this previously `canCreate` and now `canUpdate`?

Reply:
Good question! I don't think it really makes much of a difference here since PKI doesn't follow the CRUD pattern exactly. For the `pki/crl` model we don't do any creating, only updating. The `pki/config/crl` params are "created" when the engine is mounted. So any `POST` requests the UI makes to this endpoint will only ever be updating it, so we never really _create_ this model. I changed it here to try and align with what the UI is doing and keep consistence since I removed the `createRecord` method from the adapter for this model